src/service/world-api-adapter.js

```diff
@@ -226,8 +373,23 @@ module.exports = function (config) {
         },
 
         /**
-        * Get's (or creates) the current run for the given world
+        * Gets (or creates) the current run for the given world.
```

Comment:
Gets the *id* of the current run. If the world doesn't have a current run, this creates one